templates/default/agent/dotnet/newrelic.config.erb

```diff
@@ -12,7 +12,7 @@
     syncStartup="<%= @resource.svc_sync_startup %>"
     sendDataOnExit="<%= @resource.svc_send_data_on_exit %>"
     sendDataOnExitThreshold="<%= @resource.svc_send_data_on_exit_threshold %>"
-    autoStart="<%= @resource.svc_auto_start %>"
+    autoStart="<%= @resource.svc_auto_start %>">
```

Comment:
Not sure the ">" on the end is intended... Would you mind taking this out?

Reply:
I think it is needed. Without it the `proxy` element is an attribute of the `service` and not a child element. When you add it however to you to have a full `service` closing tag.

At present when a proxy is set the template creates invalid XML along the lines of:
```
<service license=".."
   attribute="xx"

   <proxy host="..."
        port="..."
   />
/>
```

After the fix, with a proxy is it should be:
```
<service license=".."
   attribute="xx">

   <proxy host="..."
        port="..."
   />
</service>
```

and when there's no proxy:
```
<service license=".."
   attribute="xx">
</service>
```

Reply:
Sorry, I meant the additional ">" at the end of the autostart line. Surely that is incorrect, no?

autoStart="<%= @resource.svc_auto_start %>"**>**

Thanks in advance!
David

Reply:
Hi David, yes the ">" is needed on line 15 as it completes the '<service>' tag. Without it you'd have

&lt;service autoStart=".."&lt;/service&gt;

and not

&lt;service  autoStart=".."**&gt;**&lt;/service&gt;

Regards,
Andrew

Reply:
Thanks for clarifying - it helped looking at the whole file rather than the files changed tab as well :-)

Merging this in ASAP!